LEAF_Request_Portal/admin/templates/mod_svcChief.tpl

```diff
@@ -292,9 +292,9 @@ $(function() {
     dialog_simple = new dialogController('simplexhrDialog', 'simplexhr', 'simpleloadIndicator', 'simplebutton_save', 'simplebutton_cancelchange');
     dialog_confirm = new dialogController('confirm_xhrDialog', 'confirm_xhr', 'confirm_loadIndicator', 'confirm_button_save', 'confirm_button_cancelchange');
 
-    $('#simpleloadIndicator').css({width: $(window).width() * .78, height: $(window).height() * .78});
-    $('#simplexhr').css({width: $(window).width() * .8, height: $(window).height() * .8});
-    $('#simplexhrDialog').dialog({minWidth: ($(window).width() * .78) + 30});
+    //$('#simpleloadIndicator').css({width: $(window).width() * .78, height: $(window).height() * .78});
```

Comment:
Do we still need this commented out code? Can we remove it?

Reply:
removed